adapters/repos/db/search.go

```diff
@@ -410,6 +411,14 @@ func (db *DB) getStoreObjects(res []*storobj.Object, pagination *filters.Paginat
 	return res[offset:limit]
 }
 
+func (db *DB) getStoreObjectsWithScores(res []*storobj.Object, scores []float32, pagination *filters.Pagination) ([]*storobj.Object, []float32) {
+	offset, limit := db.getOffsetLimit(len(res), pagination.Offset, pagination.Limit)
+	if offset == 0 && limit == 0 {
+		return nil, nil
```

Comment:
doesn't limit=0 mean to return all objects?

Reply:
This is very interesting, because I copied the existing functions, and they do the same thing.  I'll look into it

Reply:
It's working, because it does generate results, but that code does look wrong, doesn't it?

Reply:
I think we might have a default limit that is set somewhere? But not 100% sure 🤔 

What happens if you set the limit to 0 explicitly?

Reply:
```"explorer: get class: vector search: invalid pagination params: invalid default limit: 0",```

Reply:
so I guess we can never take this path anyways?

Reply:
Not from the graphql.  The function could be called from other places